site/_scss/blocks/_language-list.scss

```diff
@@ -0,0 +1,4 @@
+.language-list a {
+  color: var(--color-secondary-text);
```

Comment:
Rather than this, we can just use `class="color-secondary-text"` on the `<a>` itself. So you can actually remove this whole CSS file.